nominal/core/asset.py

```diff
@@ -35,16 +36,23 @@ class Asset(HasRid):
     _clients: _Clients = field(repr=False)
 
     class _Clients(
-        DataSource._Clients,
-        Video._Clients,
-        LogSet._Clients,
-        Attachment._Clients,
+        _DatasourceContainerClients,
         HasScoutParams,
         Protocol,
     ):
         @property
         def assets(self) -> scout_assets.AssetService: ...
 
+    def __post_init__(self) -> None:
+        """Initialize clientsbunch in DatasourceContainer abstract class."""
+        super().__init__(clients=self._clients)
```

Comment:
I think you might be able to get rid of the post init if you just swapped the order of the attribute definitions (i.e. if you put `_clients` first)